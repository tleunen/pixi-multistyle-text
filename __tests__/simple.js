it("should render a simple example correctly", () => {
	require("./support/pixi-shim");

	const MultiStyleText = require("../dist/pixi-multistyle-text");

	PIXI.settings.RESOLUTION = 2;
	let renderer = PIXI.autoDetectRenderer(200, 80);
	renderer.backgroundColor = 0x333333;
	document.getElementById("root").appendChild(renderer.view);
	let stage = new PIXI.Container();

	// Basics
	let text = new MultiStyleText("Let's make some <ml>multiline</ml>\nand <ms>multistyle</ms> text for\n<pixi>Pixi.js!</pixi>",
	{
		"default": {
			fontFamily: "Arial",
			fontSize: "24px",
			fill: "#cccccc",
			align: "center"
		},
		"ml": {
			fontStyle: "italic",
			fill: "#ff8888"
		},
		"ms": {
			fontStyle: "italic",
			fill: "#4488ff"
		},
		"pixi": {
			fontSize: "64px",
			fill: "#efefef"
		}
	});

	text.x = 100 - text.width / 2;
	text.y = 10;
	stage.addChild(text);

	renderer.render(stage);

	let canvas = document.getElementsByTagName("canvas")[0];
	expect(canvas.toDataURL()).toMatchSnapshot();
});