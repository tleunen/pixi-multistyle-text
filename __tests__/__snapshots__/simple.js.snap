exports[`test should render a simple example correctly 1`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAZAAAACgCAYAAAAisjrVAAAABmJLR0QA/wD/AP+gvaeTAAAgAElEQVR4nO3deXxU1f3/8dfsk0z2hAgJEAwSIqAEAihrrCCIggio8EMBacW2tlpLbW1dvrUudfuCtKhfsWqVvf0+vqRaFQWRnYhsQUgIW/ZtkpBlJslMZv39keSaIQmQCwrYz/Px4KG5c+fcMzdw33PPOfccTWpqqh8hhBCii7SXugJCCCGuTBIgQgghVJEAEUIIoYoEiBBCCFUkQIQQQqgiASKEEEIVCRAhhBCqSIAIIYRQRQJECCGEKhIgQgghVJEAEUIIoYoEiBBCCFUkQIQQQqgiASKEEEIVCRAhhBCqSIAIIYRQRQJECCGEKhIgQgghVJEAEUIIoYoEiBBCCFUkQIQQQqgiASKEEEIVCRAhhBCqSIAIIYRQRQJECCGEKhIgQgghVJEAEUIIoYoEiBBCCFUkQIQQQqgiASKEEEIVCRAhhBCqSIAIIYRQRQLkIhs/fjxLlixhwoQJ38vx7rvvPpYsWYLFYvlejncxPfroozzzzDOXuhpChXUTJ7LmllvOud+Ht93Gqjb/FmYmJvLltGmMjYsDwKTT8eW0abw2evR3Vlfx3ZEAuchMJhMWiwW9Xn9Ry42MjGT58uUsXLgwYLvf78disdDU1HRRj/d9CAoKIigo6FJXQ6gQYjAQYjAoP8dbLOy7+26eGzEiYL9QgwFLm/1Mej1hRiMGbfOlRwOEGY0Et9lHXDku7lVOfO+cTidutxuPx3OpqyJEO9M2bMDv93f6epPPx80ffojH5/seayUuFgmQy0R0dDSxsbE4HA6Kiorwer3KawaDQfmmrtfrlTsOj8eDw+HA6XQGlGUwGOjRowdRUVE0NDSQn5+P2+0+Zx0MBgNGoxGHw4HP5yMuLo6QkBCsVit1dXUAaLVaevfujdlspry8nNra2g7LMpvNxMfHYzabqampoays7KwXkraMRiMGgwGPx9PuziouLo6wsDDq6+spKSk57zKh+dz16NGD6OhoGhoaKCgowOVydbhveHg4CQkJ6HQ6rFYrpaWl7fYJDg7G6/XS1NSE2WymV69eeDweioqKlEC3WCzEx8cr2zv7PQQHB9OzZ0/8fj8lJSU0Njae9+cCCNbr0Wu12FwutBoNyRERmPR6TtTWUt9yTKNOx7UREei0Wk7W1WFr89m1Gg0hBgMen4/GM76MGHU6zDodDo8HdwcXepNOp9yNGHQ6woxGGj0ePD4f9k7Obyu/3x9QDwCzTodRp8PuduP3+0kMCyPGbOa008kpm63TsroFBZEQGorL6yWnpgaXhNJ3TgLkEouOjmbevHkkJycr22w2Gx999BE7duwA4Pbbb2fy5MkApKSkkJKSwttvv83+/ftpbGwMuNgkJyczb948oqOjlW0NDQ2sXbuWvXv3nrUuU6ZM4dZbb+Uvf/kLkydPJikpCQCv18s///lPTp06xYMPPkhsbCwAPp+Pzz//nH/9619KGTqdjhkzZjB27FhMJpOyvaamhvT0dPbs2XPWOiQmJvLII4/g8XhYsmSJcuHu06cP8+bNIz4+Xtm3oqKCdevWkZWVddYyAfr168f8+fPp1q2bss3hcLB27dqAOhkMBmbNmsXo0aPRar9t4c3NzeW9996jsrJS2fbiiy9itVr55JNPuP/++wkODgagqqqKxYsXc8MNNzBlyhSlOdNms/H6669TUFCglKHX65k5cyZpaWnodDoA3G43u3fvZv369e2+HHTmxZEjGd29O7M3buTlkSNJCA0FoMHj4fcZGei1Wv5r2DAiW34nLp+PF/bt45OWukQYjWy84w52lpXx6M6dAWXPSEzksZQUHs/IYHNxcbtj/2zgQOb27w/AhJ49mdCzJ4/t3s3WkhK+nDaNOpeL6Rs2dFhvs07Hzhkz2F9ZyU+3bgVgUUoKMxITmffFF/x2yBCua/N3+ZvTp3l0586A0IkwmXgqNZW0uDg0Gg0A9W43Hxw7xgc5Ofi68CVDdI0EyCUUHBzMb37zGyIjI9m4cSPHjh3DYrEwceJE7rvvPnQ6HVu3biUzMxOHw8GMGTMoKChg586dykWourqa/Px8oPlb809/+lNsNhtvvPEGNpuNq666iqlTp/LjH/+Y4uJiysrKzlmvH//4x5SWlvLee+8REhLCtGnTmDVrFvX19VitVjZs2IDRaGTy5MlMnjyZzMxMpQ7jx49nwoQJHDlyhM2bN9PY2Eh8fDzTp09nwYIF5OXlUVFR0eFxExMT+dWvfkVTUxOvvfaaUte4uDgWLVoEQHp6OkVFRURFRXH77bfzy1/+kr/85S/k5OR0+nnCwsJ46KGHsNvtvPnmm9TV1dGtWzemTp3KggULKC4upqSkBIB58+YxYsQIvv76a7Zv347X62XgwIHceuutPPbYYzz//PPY7Xal7JiYGH7yk5+wdetWioqKGDx4MMOHD+fxxx8nJCSELVu2UFBQQL9+/UhLS2POnDm8+OKLyvsXLFjAsGHDOHz4MLt378br9TJixAjS0tKIiYnhr3/96zl/X229mZbGnvJylmdl0Tc8nAXJybxwww2Y9Xq2l5ayvbSU2KAgFg4YwJPDhrG9rOycdwnn8mVJCXa3m4cGDSKrupoP8/I43smdaVcsHTOGXJuNZ1q++MxITOT66GgeGDCAJZmZABi1Wt4cN45+4eGk5+Wxs6wMo1bLPddcwy8GDSJYp+ONI0cuuC6iYxIgl9Add9xBdHQ0K1euZGebb32HDx/m2WefZdq0aezcuZP8/Hzq6uqYMWMGlZWVbN++Xdn3m2++4ZtvvgEgKSmJ4OBg0tPTlW35+flUVVXxi1/8ggEDBpxXgBQXF/PXv/4VX0sTQEhICLfddhs2m40lS5Yo230+H/feey/9+/dXAuS6667D4XDw1ltvKc01+fn5GI1GZs+eTUJCQocB0q9fPx5++GEcDgeLFy8O2Oeee+7BYDCwePFiTp48qWw/fvw4zzzzDNOnTw+4KJ/pmmuuITg4mI8++ohDhw4pdaqsrOSRRx5h0KBBlJSU0LdvX0aMGEFOTg7vvfee0jyWm5uLw+Hg7rvvZvz48QF3XBaLheXLl3PgwAEA9u/fz8CBA4mIiGDdunVs2bIFgL1799K3b18SEhIwm804nU4GDBjAsGHDyMzM5K233lKOd+jQIbRaLUOHDiUpKYnjx4+f83fW6vPCQha3XFwpKiIpPJyxcXFsLCriia++UvaLDQrinmuuYVBUFBnl5eddfkcOnz5NtdPJQ4MGUWi3sz4394LKa5Vvt/PQ9u3KHURGeTkbpkxhRMsdMMDsfv1Iiojg7aws3s7OVrZvLytj9YQJ3Ne/P2tPnqT6PO/kRNfIKKxLRKPRMGLECOrr69m1a1fAa42Njezfv5/g4OCAJptzaW1yGTNmDHEtwyQBTp06xaJFi9i8efN5lbN3714lJAClnyMzMzNg++nTpwEChhC/++67PPPMM+3a+qOiogCUZpq2kpKSlPD47//+74DwCA8P59prr+XUqVMB4QFgtVo5efIkCQkJAc1lZ2p7Xnr27Klsz8/PZ9GiRXz++ecApKamArB169Z2fSu7d+/G5/Nx3XXXBWz3eDxktl6waQ7VmpoaAPbt2xewb3V1NRqNRmnquvHGGwHYuHFju+O1fqHo39I0dL4+KywM+NnqcACwqagocHtLs2e40dil8r9Pq44fD2h+Ou10Uul0Ks1wALclJODz+1l1Rsi6vF4+LSzEoNUyuE0TmLi45A7kEgkPD8disdDQ0NBuaC7AVVddBTQ3v5yv/fv3M2rUKJKTk/njH/9IeXk5x44d48iRI2RnZ5/3SK3q6uqAn1svbp11mLe2O7fuExcXx6hRo4iPjycqKopu3boR2tImfyaLxcLDDz+M0WjE7/cHBBSgBGhUVBQPPvhgu/dHRUWh0WgIDQ3tdCjzoUOHOHLkCIMGDeLpp5/GarUGnJfWsOvevTsAhWdchKE51BsaGpQgbHX69Ol2dfb7/Xi93oCmrtbt8G2gtYb8pEmT2v1uzGYz0LXfP0BZQ0PAz60X4MqWIDmTts3v7nJT0UGdXV4v5pbQ02o0JIaF4fH7+a/hw9vt270lqCPP8uVCXBgJkEuk9RuzRqPp8CHA+vp6cnJyzmv0VCuPx8PSpUtJSUkhNTWV5ORk0tLSSEtLo6amhrfeektpajqbzi7E5zPiadKkSUyfPh1A6XPJzMzEbDZz2223tdvfYDBQXl7Ovn37mDJlCnPmzGHZsmXK663nSavVdnieqqqqqKqqCujwPpPb7eaNN95g8ODBDB06lOTkZMaNG8e4ceOora1l+fLl5ObmKsdqOOMi3Eqn07U7B539fs7nXLUez2QydXgHlZOTo9zNnC9nm9F7AfXpUimBjGc5t9+lpk4+SyuTTodWo8Hr9RLWwZ1Uo8fD1xUVyig0cfFJgFwirRepmpoaXnvttYtWrt/v5+DBgxw8eBCAhIQExowZw7hx45g6dWrAxfli69atG3feeScVFRUsXbo04E5m3LhxHb7H4XDw6quv0tDQwLXXXsugQYO44YYblJFRrefp5MmTvPPOO6rr5vP5As5L7969GT16NDfddBN33HEHS5cuVUazRUREUH5Gv0BwcDDBwcEUndEUdCHq6+uJjY1lxYoVSnPg5SjmMn3Y0+n14vL5cPt8PLRt26Wuzn8k6QO5ROrr66moqKB79+5ERES0e/2+++7jueee69IUJffccw/PP/98wLfZgoICVq9eTV1dHTExMRel7p2Jj49Hq9Vy8ODBds1gCQkJHb6nqamJ+vp6/H4/q1evxuv1MmvWLKXpprCwEI/HQ79+/TCc8bSyRqPh17/+Nb///e8DmtHONGPGDF544QWlWai13LVr11JTU6Ocl9yWzt+UlJR2ZQwePBiAY8eOnes0nLe8vDyg436O4cOH88ILLzB06NCLdryzab1zObNPRKvRcGNLc+rlxu/3c6iqijCjkWsjI9u9/uCAAXx4221c3cVmQHH+JEC+I/Hx8aSmpnb4p/X5ik2bNqHT6Zg9ezbGln+4Go2Gm2++mTFjxlBSUqJ8A299sDAyMrLT5pry8nK6devGxIkTAy6o/fr1IzQ09Lyary5Ea3NLUlKScrHX6XT86Ec/YnTLXEcddaK3KikpYfPmzVgsFmbNmgU0P2m/Y8cOIiIiuOOOO5TPpdfrmTVrFsnJyWRnZ5+1ychqtRITE8Ott94acF769u1LRESEcl4yMjJwOp1MnDgxoLM9KiqKKVOm4Ha72dryrMLFsG3bNjweD1OnTg0I98TERGbNmoXRaDzr8OSLqdHjwepwMCAqiuEto5xMOh0PX3cdfTrpv2qr9QHDq4KD0X2P/SprT5wA4DcpKQHNWJN69WLBtddS7XSSd5aHD8WFkSas78iIESMYcca8QK1ycnI4fvw4O3bsUJqYFi9eTHl5OREREYSFhVFSUsLKlSuV99jtdioqKujbty+vv/4677zzjjJ0tNWuXbsYO3YsU6ZMYdSoUVRUVBAUFETv3r2x2+2kp6d/p5+5oKCAzMxMUlJSePHFF6moqKBHjx74/X7WrFnDvffey8yZMzGbzZ2OCPv4449JTU1l2LBhfP311xw6dIj169fTvXt3Jk6cyJgxY6isrCQ2NpagoCCysrL45JNPzlqvjIwMxo0bx+TJk7nxxhuxWq2YzWYSEhJoaGhQzkttbS3vv/8+DzzwAE899RSFhYV4vV569+6NVqtl5cqVAQ8SXiir1cr777/P/fffz/PPP09ZWRl6vZ5u3brhdDp58803u/xE+oV4NzubJ1JT+Z+0NOpcLoL0etw+H+8dPcoDAwac9b1VTielDQ0M7daNnTNm8IevvmJry7M136XtpaX8LTubhQMG8NmUKeTb7USYTMQGBVHa0MCT53hwVVwYXVxc3DOXuhI/JE6nk6KiIg4fPtzpn+zsbGWo6jfffEN+fj5arRatVktZWRlbt25lzZo17TqzDx8+jN1uJz8/n6NHj7br7PX7/Xz99dc0NjYq05/U1NRw4MABVq5cqUxHcra6FxYWcurUqYApPlwuFyUlJZw4cYL6+nplu8fjUUY0tY7QOnDgAHa7Ha1Wi9vtZt++faxYsYITJ05QUVGBy+Xi+PHjVFVVUV9fz8mTJylu83Sz1+vlxIkTFBUV0dTUhNVqxev1smfPHiorK9Hr9Wg0GgoKCtiwYQMffvhhu1FQZ/L7/ezbt4/GxkZ0Oh3BwcHU1NRw8OBBVq1aFdDcVl5ezsGDB/H5fFgsFtxuN4cPH2b16tVkt3nOAJqfLM/Ozm7XX2K32zl27Fi70VyNjY2cPHmS3Nxc5Y6ytLSUAwcO4PP5MBqN2Gw29u7dy8qVK5WHG8+HzeXiYFUV2TU1AUNf691usqqrOVJdjaPNSC+H18ux2loyT5+mruV3fbSmhgNVVTR6PFQ6HGwpKeFPe/eSa7ORZ7ORWVWFvaVDurqpia+tVmVqET/NF/PapiayqqvZXV6O3e3mdFMTe9rs5/B4OF5bq5Tlp3mE2B6rlYKWUWv1Hg9HTp/mSHU1rjM60mtajnuizd/l/ZWV7K2sRKvRYNBqKW5oID0vjz/v389pef7jO6VJTU2V5/yFEEJ0mfSBCCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIcQFCw8PJyQk5FJXQ3zPJEBEO/fffz/Lly8n6DKchTVs4jpCb1lzzv3Cb/uQsAmrvocaBTrf+v1QXH/99SxZsoRXXnmFqVOnXurqiO+ZBIi4omgMIWgM337T1Vriibx7H5YRz52xXygYzn8m43OVp7Z+F8uTTz7J66+/ftHLPVNMTAzLly/ngQceOK/9b7/9diwWC19++WXAyoziP4NMpih+kOo2TIPzWNRJXJioqCicTif/+Mc/LnVVxCUgAXKF0Ov1xMfHExoais1mo7S0tN0yqEajEYPBQGNjI36/nx49ehAeHo7dbu90Yj6NRkP37t0xm81YrdYuz/6q0Qej0erxuWyg0aKLSEajN+GtPYHf3TzxokZnRBdxLWh1+OpONu+rHF+LxhCC3+fB7wk8tkZnRKMz4/c48Ps6WFVOZ/r2277OgNYYhs/TCD4Pfpe9/f6A1hyFNvRqNAYL3vpifLbcc5an1Tcvjdq23mfW0edtAm/HKzkqxw7qhjY0AbwuPDU54HOddX9oXokxKChImcLfYrHg8XjaTbQZExNDt27dcDgcFBYWBkwwqdfrMZlMeL1enGdMLmgwGNDr9TgcDmUCztb3WCwWnE6nMvFjZ/XrbAVHk8lE3759CQoKoq6ujry8vHZlmUwm9Ho9jY2NaLVaevfujc1mu6wX2BLfkgC5AkyYMIFbb701YF3xhoYGvvjiCz799FNl28yZM7npppt46aWXuPvuu+nbt6/yWl5eHsuWLQv4x56YmMiCBQuIbVn/wev1snHjxrMuznQmy8gXMXQfjW3jbEJGvtx8gQT8ngYaMn4PWj3Bw/4LrallwR+fi4Z9L+AqaJmC3RhB+B0bcZftpH7nowFlGxNnEJzyGPUZj+Mubj/9e9DAn2HuP7d5354TMPacQP3ux3CXbCVi2pf4XHXYNjQvr4tGQ9B1D2Pq9//QaL9dmMpz+jANGY/jc1R0Wl7QgAfRRSRh+/xuvLa8wDpc/2tM19xN/VdP4C7a2OE50pgisAx7GkOPcdBybv0uO86c92k6vhK/v/PZhPv06cPjjz+u/LxkyRK++uor/v73vwMQHR3N/fffr6wxAyhT9+/atQtoXmHx6aefxmAw8Oc//1mZ/Vij0fDLX/6S5ORk1q5dS3R0NBMnTgRgyJAhDBkyhOXLl7dbNgBg1KhRzJ8/H4CQkBCWL1/O1q1bWbt2LQBpaWnMnDkzYHGzmpoaVq1axZEjR5Rtc+bM4cYbb+Tpp5/mwQcfpFevXqxfv57PP/+803MiLh8ynftlbtCgQSxYsICKigpWr17Npk2byMnJoXfv3gwfPpyioiKsVisA1113HX369OH666/Hbrfz8ccfc/DgQcLCwkhMTMRgMJCVlQU0Nz389re/JSwsjG3btrFp0ybKysqYMGECPXr0QK/X89lnn7W7yzmTMWEyupDeGHvejKcyE0fO3/HZ8zHEDsfQYyymhMm4rV/RlPM+nuosDN2GYowfR9Opf4LXhUYfhLn/XHz1hbgKPwsoWx89CEP3UbiKv8DXcuE2J80B/DSdWAueRvwNpehjh+OtzsKZ/Q7e04fwu+2Yk+fj9zbRdLK5acXUbzZBg36OK//fNB58BXfBJ3gdlZh63YI+aiCu/H93Wh5+L4arbsTXVI2nss3FVKPFMuxpNEDjvufB7wmsH4DWSOiP/oYhZihNeetpyn4Hd8kWtJY4TInTQKPDU7G30/Pr9XqprKykZ8+emEwm1qxZQ05ODqdPnyYkJIQ//OEPxMXFsWnTJj799FOys7OJj49n7Nix1NXVUVhYSGNjI42NjQwePJirr76aXbt24ff7GTt2LDfffDNHjhzhf//3f3E4HFRVVZGcnExBQQGffPIJJ06cwOFwtKtXU1MTZWVlJCUlYbfbWb9+vbKG++jRo7nvvvsoKSlh9erVbN68mby8PJKTkxk7diwnTpxQ7jCGDBlCz549SU5OJiQkhMzMTLKysqiqqjrr3ztxeZA7kMtc61Kqb7/9trKGSEFBAQ0NDTz66KP06dOHQ4cOBbzHarWyZMkSZZW+7OxsXn75ZZKTk5V9pkyZQnBwMGvWrGFbm/Wks7Oz+d3vftflejYVfo4jczEA7iLQhSdhiBuLu2gjDV89oeynCYrFfM096KMG4S7P6PJx2vKcPozPWY150EN47YU05a7vdF9DXBp+vw/HgZfx+9x4AXflAbSmCAw9J6Axhndanr/wc8zXPYwxfjzO7G/XZdfHpKAxR+PK/xi/t+N1J8xJc9BHJOHMWo4j+2/KdnfZdkJvWYMp6T6cJ9bgb6rt8P02m43t27czduxYQkND2b59u/La1KlTiYyMZMWKFcrdBjSvG/OnP/2JO++8k127duHz+dixYwcpKSkMHDiQSZMm8dVXX3HXXXdRX1/PihUr8Pv95OXlYbfblXXt2x7rTBUVFVRUVDBt2jQaGhqUfQ0GAzNnzqSxsTFgnfn8/HwKCgp46qmnmDlzJi+++GJAeS6Xi1deeaXDsBKXLxmFdZn797//zZNPPqmER6vWddQ7Wt528+bNAUu82mw2qqurlSYwjUbDsGHDqKurY8eOHQHvzc3NVbX07Zl3Dz5H812Rq2hTwHZ/Y/N2jTG8y8e4EBqNFo1Gi3nAwoBRUo37/0zdhzfjd3W+2JbPUYGnYh+6iH5oQ3or2409JwDQVPBpZ2/FlHAb+H04j60M2O73unAVfIpGZ0QfPbjLn0er1TJ8+HBsNhu7d+8OeK2hoYHMzExCQkKUpXn9fj8rVqygsbGRKVOm8OCDD2I2m1m9evU5Fxrriv79+2OxWJSFzdoqLi6moKCAhISEds+MfPLJJxIeVyC5A7nM2Ww2oqKiGD9+PAkJCURHRxMTE6MESEfarrDXyuv1KuuRx8bGYjKZyMvL63A1P6vVytVXX92levoaygI3tLTr+xwdLwGr0Xy/310c3ywjJO0NzNf+GHP/+/BUfYO7Yg/uku14bafO+X534QYMV43A1GsCjqPvodFoMfS8GZ+jEk/lvo7fpNGhCe0DPjfBI55p97I2uHvzf81RXf48kZGRWCwWGhsbWbhwYbvXu3dvLjssLEzZVltby+rVq1m4cCGJiYlkZGR02L9xIeLi4oDmu+SOVFVV0adPHyIjIwNWtywrK+twf3F5kwC5zA0dOpQFCxZgNBqxWq2UlJSQk5ODzWZjzpw5Hb7H7e5gxFIbRqMRoNNvfG2Xsz1vnTTh+FE/lFajNap+75k81Ueo+3Q6pj5TMfQYgy5mMPrYYQQN+gWuki00fPUH8HXe3+Mq/pLgob/H0HM8jqPvoYsZjNYc03xn0VknuM6ERqPF7/ehNYa1f93TiKfi605HjJ1Na+e03+/HYmn/vIvdbicnJ6fd77Lt6K3v4ht/a73OHO11Ln4Zcn1FkgC5jOn1eubPn4/H42Hx4sUBTUttR910VetIrKiojr/5RkdHqy77YtIGxVzU8vxN1TiPfYDz2Ado9EHou4/CfO1PMMb/CHfcOFzFX3b+Xk8DrtJtGHtNRBvSU2m+chV23nyF14Hf6wKfC/u2hy7qZ7G3rB9eU1PDa6+9dl7vsVgszJ07l6amJux2OzfddBOHDh0iJyfnotWr9e9W2zuftmJimn+nNTU1F+2Y4tKRPpDLWHR0NGazmZMnT7brl+jVq5fqcqurq6mtraV3797twsJisXDNNdeoLrurWjufz+wT0Wi0GK668aIcQ6MzEzY5naDBi749rseBu3gzzqz/AUAbHHfOclyFGwAw9roFQ/zNeGqP46090fkb/H681YfRGMPRRfRv93LQwJ8RftuHytDnrrDb7VRWVtK9e/cOmzPnzp3Ls88+G3B3cu+99xIeHk56ejrvv/8+APPnz7+oU9acOtXcHDhkyJB2r0VGRtK7d2+Ki4sDmq/ElUsC5DJWV1eH3++nV69eAR3gKSkpTJs2DWi+S1Hjyy+/RKPRMHv2bMxmM9A8gmbevHkBY/e/c55GfA4r+qgBGGKHN2/TmTBf93Bz/8G5tDxgqA2+CjS6Dnfxe53gbsCUOA1dWJu+HY0WQ/zNAHhrss9Znrs8A39TLaakuWiDuuE+S+d5K+fx1QAEpzyGxvjtczzG3rdiSp6P33kan73j/oKAY7vd6HS6gLDYvHkzer2e2bNnYzA0P9ui0WgYP348o0ePpqioSLkjGDFiBKmpqZw6dYpt27Zx4sQJtm7dSlRUFLNmzQo4DjQP0uhogMa5FBQUKEN2R44cqWzX6/XMmjULrVbL5s3tn+kRVyZpwrqMOZ1OPvvsMyZPnjc6SY8AAAjGSURBVMwLL7xAcXEx0dHRhISEsG7dOu666y7S0tLQaDT885//7FLZGzdupG/fvgwePJhXXnkFq9VKbGwsTqeTPXv2cMMNN3xHn6o9Z/a7BKc+QUja/+B31aHRB+H3uWk6+h7mAWefk8nvrMLbUIq+21AiZuyk4as/4C7Z2m6/xkOLCRn3JmG3rMVrO4XPWd08qsocg6vgE9wtz3ectTyfh6aijZivuQe/z3325qsW7tLtOLPfwTzgAcKnfI7Pno/GFIE2KBZvQykNe546r3OUn59P3759efnll9m1axcrVqxg69at9OrVi9GjR7N48WKsViuRkZGEhoZSVFTEmjXNkzpGRkYye/ZsPB4PK1euVAZO/Otf/+L6669n5MiRZGZmkpmZid1u5/Tp0/Tr149ly5bx7rvvdrmj/e9//zuLFi3i/vvv5/bbb6empoaePXsSHBzMrl27yMi4sOHb4vIhDxJe5nJycrBarWg0Gnw+H9nZ2axcuZKjR4+Sn5+Px+MhPz+f4uJinE4n+fn55ObmtutIt9vtHDt2jKKiImXb/v37qa6uxmAw4HK5yMzMZMWKFZSVlVFYWEhBQUGHo7Ta8rlseKsO4qnJDuhM9rvr8VZn4a0+gt/TprPW68BbewzP6Uxl6Ky35iieqgP4PY34HZXNndp7/4TXlovPloenKhO/u7nN399Ujcf6dZuRU37cpdvxN9Xiqc7CU74bv9uOr+k0busefC37+RrLcZc1D1nWGCyg1eOtyabp+Gqajr4LSmd/x+V9+4GbMPWZgrt4M678j9udj/b1A0/lPryV+5qf8Nca8NcX4cr7F437/4zPeX5Tdhw/fpy6ujpKS0vJyspShnUfOnSIwsJCtFotWq2W0tJStmzZwrp165QO86uuuory8nK2b99Obu63U7d4vV6OHTtGcXExLpeL8vJy/H4/mZmZ1NfXk5+fT1ZW1lmnt7HZbOTk5ASMompoaCAjIwOHw4HZbEan05Gbm8uHH37Ipk2Bw7odDge5ubnk5uaedcoUcXnSpKamyvAHIc6T+doFBA36BfZtP8NT0cnwXSH+Q0gfiBDn0jJ/lcYYirnv3Xhrj+Op3H+JKyXEpSd9IEKcg9YQStjkdDQ6M+hMOPb/WaaKFwIJECHOyedtwl22G7/Xgac8A3fZzktdJSEuC9IHIoQQQhXpAxFCCKGKBIgQQghVJECEEEKoIgEihBBCFRmFJa4YqampPPVUx1N/NDU1UVhYyObNm9m4caMyPfhNN93Er371K95++202bNjQpeOtXLmS+vp6fv7zn59zv5qaGh555JEulQ8wffp05s2bx//93/+xatWqLr9fiEtJAkRcMXQ6HaGhodTU1JCXlxfwWmRkJCkpKQwdOpQ+ffqwfPlyoHntk9DQUGUNlK4ICQk551Qurfu1XWejKy6kfkJcahIg4oqTmZnJCy+80G57UlISixcvZubMmXz00UeUlZWxZcsWMjIyVF3g586de14LHc2dO/e8gkaIHxrpAxE/GMePH2fXrl1otVoGDhwINE9PbrfblZX5LBYLoaGhyvTnbQUHBwfcCdTX1yvToZ9NfX19hxMORkZGMnjwYEaOHElCQtfX/BDicicBIn5QzlxKdeLEiXzxxRfK+ilz5swhPT2dZ599NmC//v37k56ezooVK5S1V9LT0/nggw/Oecz09HTeeecd5WeNRsPChQtZs2YNixcv5rnnnuPdd99l2bJlyop8QvwQSICIHwyTyURKSgpAwLTlbX3wwQfk5+czfPhwbrnlFqB5Ia3HHnsMrVbLq6++qiwXq9add97JrFmz2LRpE4888giLFi1izZo1JCUl8cQTTwTs23qHc7Yp04W4XEkfiLjixMbGkpaWFrAtOjqaSZMm0bNnT/bu3dtpgLhcLl566SWWLVvGQw89xL59+5g2bRpXX3016enp7N9/4bPsjho1Cp/Px7Jly5R1Wb755hvCwsJIS0sjLCwMm80GQG1tLYDysxBXEgkQccUZOHCg0sdxpqysLF599dWzvv/kyZOsWrWKBQsW8Mwzz9C/f38KCgr429/+dlHq17q409y5c/nHP/6h3GUsXbqUpUuXBuxbV1cX8F8hriQSIOKKk5WVxfr16wO2uVwuSktLKSg49/riAOvWrePGG29k4MCBuN1uXnrpJaWj/UK98847vPzyy8yZM4e77rqL7Oxs9u/fT0ZGBvn5+QH7tt6BSICIK5EEiLjiVFRUsG3btgsqw+fzKR3ubf//Yjh69Cjz589n0qRJ3HDDDQwcOJCUlBR+8pOfsHPnTp5//nk8Hg/wbXBIE5a4EkknuviPdOeddzJkyBByc3MxmUz87ne/Q6fTXbTya2pqWLduHb/+9a+ZPn06zz77LKdOnWLMmDGMHDlS2a++vh6v1ysBIq5Ish6IEEIIVeQORAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKGKBIgQQghVJECEEEKoIgEihBBCFQkQIYQQqkiACCGEUEUCRAghhCoSIEIIIVSRABFCCKHK/wdPy9EDjnttYgAAAABJRU5ErkJggg=="`;
